Core.ApplicationServices/Contract/ItContractService.cs

```
@@ -221,18 +221,20 @@ public Maybe<OperationError> TransferPayments(int contractId, Guid targetUnitUui
 
         public Result<IEnumerable<DataProcessingRegistration>, OperationError> GetDataProcessingRegistrationsWhichCanBeAssigned(int id, string nameQuery, int pageSize)
         {
-            if (string.IsNullOrEmpty(nameQuery)) throw new ArgumentException($"{nameof(nameQuery)} must be defined");
```

Always check your input parameters? Fail fast, is one good reason?